go.mod

```diff
@@ -4,6 +4,7 @@ go 1.15
 
 require (
 	github.com/container-storage-interface/spec v1.3.0
+	github.com/evanphx/json-patch v4.9.0+incompatible
```

Comment:
Used in kubernetes/kubernetes:
https://github.com/kubernetes/kubernetes/blob/master/go.mod#L42